### PR TITLE
Added installation instructions for new pkg mode in README

### PR DESCRIPTION
**Pull request by rahulkp220**
How to install/update using the `pkg` mode/prompt.
```
julia>] # switch to pkg> mode
(v1.0) pkg> 
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,27 @@ A very simple package for accessing elements in the Periodic Table! :fire:
 
 ## Installation
 Since PeriodicTable is registered in `METADATA.jl`, you can directly install it like,
-```jl
+```julia
 julia> Pkg.add("PeriodicTable")
```
Comment (carstenbauer) [inline README.md]: Add `using Pkg` here.

File: README.md [hunk 2]
```diff
@@ -13,19 +13,27 @@ A very simple package for accessing elements in the Periodic Table! :fire:
...
+(v1.0) pkg> add PeriodicTable
 ```
 
 ## Update
 ```julia
 julia> Pkg.update("PeriodicTable")
+
+# OR
+julia>] # switch to pkg> mode
+(v1.0) pkg> update PeriodicTable
```
Comment (carstenbauer) [inline README.md]: I'd drop this section.

**Review by carstenbauer — CHANGES_REQUESTED** [review]
I guess it's a bit inconsistent in it's current form. To use `Pkg.add` I have to `using Pkg`. So the upper suggestions only works on 0.6 and the lower one only on >= 0.7.

In my opinion, we should only write it for the 1.0 world. This is the major audience and people from pre 1.0 time are likely to know how do add the package anyway. Concretely, I'd just add a `using Pkg`.

Personally, I'd also drop the whole "Update" section. It seems irrelevant to me.

**Comment by carstenbauer** [comment]
Slightly OT, but I also thought about reversing the order in the `How it works` section. Wouldn't it be better to have the full PT first and then explain how to access detailed element data?

One more thing (sorry :D), the "Data by" section should probably get a sentence that only parts of the data come from the JSON source. Some information has (and will be) added.

**Comment by rahulkp220** [comment]
I guess the `update` section was always a carryover from the time this package was created, and I agree, we can have it removed as for someone who knows how to add a package, it would be too naive not to know how to update.

File: README.md (hunk 1 above)
Comment (rahulkp220) [reply README.md]: Would this seem better? Later when we drop support for version `0.6`, we will have this part removed.
```julia
# for version <= 0.6
julia> using Pkg
julia> Pkg.add("PeriodicTable")

# for version >= 0.7
julia>] # switch to pkg> mode
(v1.0) pkg> add PeriodicTable
```

Comment (carstenbauer) [reply README.md]: No, the upper one won't work on 0.6 (there exists no module Pkg on 0.6). As I said, I'd just drop the `# for version ...` comments and only give instructions for 0.7+.

If you would really want to indicate both ways (0.6 and 0.7+) then I'd switch the order (0.7+ first) and remove the `using Pkg`.

File: README.md [hunk 3]
```diff
@@ -75,30 +84,15 @@ julia> elements[1:4]
...
-Uue                                                   
-      La Ce Pr Nd Pm Sm Eu Gd Tb Dy Ho Er Tm Yb Lu    
-      Ac Th Pa U  Np Pu Am Cm Bk Cf Es Fm Md No Lr
-```
-
-## Data by
-* [Periodic-Table-JSON](https://github.com/Bowserinator/Periodic-Table-JSON)
+### Data by
+The data used for this package has been pulled up in parts from [here](https://github.com/Bowserinator/Periodic-Table-JSON)
+Some information has been (and will be) added over the period of time.
```
Comment (carstenbauer) [inline README.md]: "Some information has been (and will be) added over time." sounds better to me.

Also there seems to be a dot missing after the `here` link.

File: README.md [hunk 4]
```diff
@@ -11,32 +11,41 @@
...
+Cs Ba    Hf Ta W  Re Os Ir Pt Au Hg Tl Pb Bi Po At Rn
+Fr Ra    Rf Db Sg Bh Hs Mt Ds Rg Cn Nh Fl Mc Lv Ts Og
+Uue                                                   
+      La Ce Pr Nd Pm Sm Eu Gd Tb Dy Ho Er Tm Yb Lu    
+      Ac Th Pa U  Np Pu Am Cm Bk Cf Es Fm Md No Lr
 ```
-exports a global variable called `elements`, which is a collection of
+
+In particular `PeriodicTable` exports a global variable called `elements`, which is a collection of
 `Element` data structures.  You can look up elements by name (case-insensitive)
```
Comment (carstenbauer) [inline README.md]: I'd move the first sentence, "In particular `PeriodicTable` exports a global variable called `elements`, which is a collection of `Element` data structures. ", before the PT visualization.

**Review by carstenbauer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rahulkp220) [reply README.md]: I missed that part.

**Comment by rahulkp220** [comment]
Hopefully should be enough for now.

File: README.md [hunk 5]
```diff
@@ -11,25 +11,37 @@
 # PeriodicTable.jl
 A very simple package for accessing elements in the Periodic Table! :fire:
 
-## Installation
+### Installation
 Since PeriodicTable is registered in `METADATA.jl`, you can directly install it like,
-```jl
+```julia
```
Comment (carstenbauer) [inline README.md:16]: In the future we might have to replace `METADATA.jl` by the general registry on https://github.com/JuliaRegistries/General.

**Review by carstenbauer — APPROVED** [review]

**Pull request merged**